src/api/AstroImageAPI.ts

```diff
@@ -0,0 +1,11 @@
+
+
+export async function getAstroImageUrl() {
+
+    const url = "https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY";
```

Comment:
We should use a personal api key here instead of `DEMO_KEY`, see this article for how we want to store environment variables like this to protect them and not commit to source control.

https://dev.to/khalidk799/environment-variables-its-best-practices-1o1o

On this occasion we don't want to put the `.env` file in the gitignore as we are calling the API directly from the app running on the clients browser - this is OK-ish, as the key is only used for rate limiting.